force-app/main/default/classes/BDI_AdditionalObjectService_TEST.cls

```diff
@@ -1371,6 +1378,7 @@ private class BDI_AdditionalObjectService_TEST {
         }
 
         bdiDIS.process(null, dis, diRecords);
+        Test.stopTest();
```

Comment:
@force2b Could you provide a brief explanation what was going wrong here in this test and why using startTest / stopTest resolved it?